data_encryption/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_encrypted_data,access_encrypted_data,model_encrypted_data,base.group_system,0,0,0,0
```

Comment:
system users cannot read them?

Reply:
No. This table is meant to contain sensitive and diverse data.
So probably no user should be able to access the whole table. 

When you implement a submodule, you should call the _get/_store methods with sudo.
So it is the rôle of each submodule to implement who is able to read or store data, but by default, nobody can except the superuser.
So you could have a module that allow to store bank passwords for instance
Another carrier password, and a system user may be able to access only the carrier stuff but not the bank stuff..